code/modules/admin/verbs/cinematic.dm

```diff
@@ -1,21 +1,16 @@
-/client/proc/cinematic(cinematic as anything in list("explosion", null))
-	set name = "cinematic"
-	set category = "Debug"
-	set desc = "Shows a cinematic."	// Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
-	set hidden = 1
+/client/proc/cinematic()
+	set name = "Cinematic"
+	set category = "Event"
+	set desc = "Shows a cinematic." // Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
+	set hidden = TRUE
+
 	if(!SSticker)
 		return
+	if(!check_rights(R_EVENT))
+		return
 
-	if(!check_rights(R_MAINTAINER))
+	var/datum/cinematic/choice = input(usr, "Choose a cinematic to play to everyone in the server.", "Choose Cinematic") in sortTim(subtypesof(/datum/cinematic), GLOBAL_PROC_REF(cmp_typepaths_asc))
```

Comment:
Can this input be cancelled without making a choice, or will you be forced into choosing something if you misclick this verb?

Reply:
Can't be cancelled, but there is far_explosion, which is just creating a sound